packages/docs/.vuepress/config.js

```diff
@@ -0,0 +1,79 @@
+// .vuepress/config.js
+module.exports = {
+  theme: '@okta/vuepress-theme-nimatron',
+  dest: 'dist',
+  head: [
+    [
+      'meta', { 
+        'og:title': 'Odyssey, the Okta design system',
+        'og:type': 'article',
+        'og:url': 'https://design-docs.trexcloud.com',
+        'og:image': 'https://www.okta.com/sites/default/files/okta-logo-onblue.png',
+      },
+    ],
+    [
+      'script', { 
+        'src': 'https://unpkg.com/micromodal@0.3.2/dist/micromodal.min.js',
+      }
+    ],
+    [
+      'script', { 
+        'src': 'https://unpkg.com/choices.js@9.0.1/public/assets/scripts/choices.min.js',
+      }
+    ],
+  ],
+  // These plugins include vendor libraries which are required by Odyssey docs. 
+  plugins: [
+    require('./plugins/plugin-choices/index.js'),
+    require('./plugins/plugin-micromodal/index.js')
+  ],
+  themeConfig: {
+    nav: [
+      {
+        title: 'Getting Started', // required
+        children: [
+          { title: 'UI Status', link: '/getting-started/status', isVisible: true },
+          { title: 'Changelog', link: '/getting-started/changelog', isVisible: true},
+        ]
+      },
+      {
+        title: 'Base',
+        children: [
+          { title: 'Color', link: '/base/color', isVisible: true },
+          { title: 'Elements', link: '/base/elements', isVisible: true },
+          { title: 'Design Tokens', link: '/base/Tokens', isVisible: true },
+          { title: 'Type', link: '/base/type', isVisible: true },
+          { title: 'Grid', link: '/base/grid', isVisible: false },
+        ]
+      },
+      {
+        title: 'Components',
+        children: [
+          { title: 'Banner', link: '/components/banner', isVisible: false },
```

Comment:
Might be a good idea in the future to automate this by walking through our doc `components` directory.

Reply:
I agree, this is something I plan on revisiting as part of the next set of work related to docs site redesign.